test/univariate/continuous/noncentralbeta.jl

```diff
@@ -0,0 +1 @@
+test_samples(NoncentralBeta(3,2,1), 5000; rng=MersenneTwister(1234), nbins=50)
```

Comment:
This is already tested (in a better way) in https://github.com/JuliaStats/Distributions.jl/blob/35125579a77b938f8d4a8bd2be23cbf1f2ddf225/test/univariates.jl#L152